meson.build

```diff
@@ -38,28 +38,14 @@ has_backends = false
 # Third party files.
 includes += include_directories('third_party', is_system: true)
 
-# Both protobuf and protoc must be the same version, so couple them together.
-protobuf_lib = cc.find_library('libprotobuf', dirs : get_option('protobuf_libdir'), required : false)
-if not protobuf_lib.found()
-  protobuf_dep = dependency('protobuf', required : false)
-else
-  protobuf_dep = protobuf_lib
-endif
-protoc = find_program('protoc', required : false)
-# For tensorflow skip system protobuf, chances are it will not work.
-if get_option('protobuf-3-6-0')
-  protobuf_dep = subproject('protobuf-3.6.0').get_variable('protobuf_dep')
-  protoc = subproject('protobuf-3.6.0').get_variable('protoc')
-elif not protobuf_dep.found() or not protoc.found() or get_option('tensorflow')
-  protobuf_dep = subproject('protobuf').get_variable('protobuf_dep')
-  protoc = subproject('protobuf').get_variable('protoc')
-elif protobuf_lib.found()
-  includes += include_directories(get_option('protobuf_include'))
-endif
-deps += protobuf_dep
-
-gen = generator(protoc, output: ['@BASENAME@.pb.cc', '@BASENAME@.pb.h'],
-  arguments : ['--proto_path=@CURRENT_SOURCE_DIR@/libs/lczero-common', '--cpp_out=@BUILD_DIR@', '@INPUT@'])
+# Compiling protobufs.
+python = find_program('python3')
+gen = generator(python, output: ['@BASENAME@.pb.h'],
+  arguments : [
+    '@CURRENT_SOURCE_DIR@/scripts/compile_proto.py',
+    '--proto_path=@CURRENT_SOURCE_DIR@/libs/lczero-common',
+    '--cpp_out=@BUILD_DIR@',
+    '@INPUT@'])
```

Comment:
I would rewrite this as:
```
compile_proto = find_program('scripts/compile_proto.py')
gen = generator(compile_proto, output: ['@BASENAME@.pb.h'],
  arguments : [
    '--proto_path=@CURRENT_SOURCE_DIR@/libs/lczero-common',
    '--cpp_out=@BUILD_DIR@',
    '@INPUT@'])
```
From the meson docs:
>Meson will also autodetect scripts with a shebang line and run them with the executable/interpreter specified in it both on Windows (because the command invocator will reject the command otherwise) and Unixes (if the script file does not have the executable bit set). Hence, you must not manually add the interpreter while using this script as part of a list of commands.